backend/routes/EventRouter.js

```diff
@@ -0,0 +1,89 @@
+const express = require('express');
+const router = express.router();
+
+//Get all events
+// router.get("/");
+router.get('/', async (req, res) => {
+    try {
+      const result = await pool.query(`
```

Comment:
Pool here isn't defined. To access the database, you need to import `database.js`.